packages/test/jest.test.js

```diff
@@ -27,12 +29,49 @@ function compareTimestamps(ts1, ts2) {
   return expect(Math.abs(ts1 - ts2)).toBeLessThan(1e2);
 }
 
-let entries, id1, id2, id3;
+let entries, id1, id2, id3, message;
+
+beforeAll(() => {
+  websocket.onmessage = event => {
+    message = jrp.parse(event.data);
+  };
+});
+
+afterAll(() => {
+  websocket.close();
+});
 
 test("create entry", async () => {
   id1 = await call("createEntry", { name: "name1", content: "content1" });
+  expect(message.type).toBe("notification");
+  expect(message.method).toBe("createEntry");
+  expect(message.params.entry).toEqual({
+    id: id1,
+    name: "name1",
+    content: "content1",
+    created: expect.any(Number),
+    updated: expect.any(Number),
+  });
```

Comment:
You could probably simplify the test by comparing with the result of `listEntries`.